src/azul/plugins/__init__.py

```diff
@@ -428,15 +428,22 @@ def special_fields(self) -> SpecialFields:
         """
         raise NotImplementedError
 
+    @property
+    def root_entity_type(self) -> str:
+        """
+        The type of entity that sits at the root of the entity graph, containing
+        entities of all other types.
```

Comment:
```suggestion
        The type of entity that sits at the root of the entity graph, and that all other entities are directly or indirectly associated with. Typically, entities of other types are thought of as *belonging to* the root entity and this relationship is implied rather than made explicit via a foreign key or some other manifestation of a graph connection. The mere presence of a `project` entity in a TDR snapshot for HCA, for example, implies that all other entities in that snapshot *belong* to that project.
```